Cookbook/Technical-Documents/TechnicalDebt.md

```diff
@@ -7,111 +7,246 @@ Technical debt and legacy code has a sad tendency to build up in any software pr
 
 GatewayManager supplies global dependencies to legacy Objective-C code. There was plenty of code in version one and two of the Babylon app that relied heavily on singletons.
 
-| Objective-C view Controllers | Location | Comments |
-| ---------------------------- | ---------| -------- |
-| BillingInformationViewController | Clinical Records | CNSMR-930 |
-| EditMembershipCodeViewController | Clinical Records | CNSMR-934 |
-| MembershipTableViewController | Me Tab | CNSMR-926 |
-| SurveyViewController | Clinical Records | CNSMR-937 |
-| InsuranceViewController | Clinical Records | CNSMR-942 |
-| BBAddAdditionalPatientInformationViewControllerV2 | Clinical Records | Might be dead code but will be deleted as part of CNSMR-947 |
+| Objective-C view Controllers | Location | JIRA Tickets | Comments |
+| ---------------------------- | ---------| -------- | -------- |
+| BillingInformationViewController | Clinical Records | CNSMR-930 | |
+| EditMembershipCodeViewController | Clinical Records | CNSMR-934 | |
+| MembershipTableViewController | Me Tab | CNSMR-926 | |
+| SurveyViewController | Clinical Records | CNSMR-937 | |
+| InsuranceViewController | Clinical Records | CNSMR-942 | |
+| BBAddAdditionalPatientInformationViewControllerV2 | Clinical Records | | Might be dead code but will be deleted as part of CNSMR-947 |
+
+Bento versions that use the design library are currently (July 2019) being worked on. Some have been completed but are still under feature switches.
 
 Some additional information about [BBAddAdditionalPatientInformationViewControllerV2](./BBAddAdditionalPatientInformationViewControllerV2.md)
 
 ## Free Standing Swift View Controllers that should be Moved to Bento.
 
 | View Controller | Comments |
 | --------------- | -------- |
-| InfoViewController | Could be replaced with a Bento Component for scrolling through a couple of info screens. Only used for the GP @ hand on-boarding so could be moved from BabylonUI. |
+| InfoViewController | Could be replaced with a Bento Component for scrolling through a couple of info screens. Only used for the GP @ hand on-boarding so could be moved from BabylonUI. Used when `isV4IDVerificationEnabled == false`|
 | SegmentedViewController | Currently only used in Healthcheck to show the the two bands with organs and layers. Should probably be moved from BabylonUI. |
```

Comment:
`AvatarViewController` is also a free standing ViewController, but I'm not sure how valid it would be to replace with Bento. SceneKit and the complex management of state could cause issues.

Reply:
I agree that it is unlikely that we will ever want to port `AvatarViewController` to Bento.